browsers/firefox/manifest.json

```diff
@@ -6,6 +6,7 @@
             "strict_min_version": "91.0"
         }
     },
+    "default_locale": "en",
```

Comment:
What's the implication of that? Will Firefox builds now get localized name + description where they didn't before?